pages/search/formatters/tasks.jsx

```diff
@@ -18,14 +18,22 @@ export default {
       const highlight = get(result, `highlight.licenceNumber[0]`);
       const licenceNumber = highlight ? <Markdown>{highlight}</Markdown> : get(result, 'licenceNumber');
 
-      if (model === 'pil' || model === 'trainingPil') {
```

Comment:
Wouldn't a switch case make it easier to add new models?

Reply:
Agreed - I've done some refactoring.